code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -40,7 +40,7 @@
 	. = ..()
 	if(!. || !client)
 		return FALSE
-	to_chat(src, "<span class='notice'>You can kill and eat other drones to increase your health!</span>" )
+	to_chat(src, "<span class='notice'>You may resume your mission, operative!</span>" )
```

Comment:
Just remove the line. Login() is called whenever a player starts controlling a mob, not just when (re)connecting to the server.

Reply:
Ah, gotcha